Query/events/base_event.json

```diff
@@ -0,0 +1,17 @@
+{
+  "caption": "Base Event",
+  "description": "The base event is a generic concrete event and it also defines a set of attributes available in most event classes. As a generic event that does not belong to any event category, it could be used to log events that are not otherwise defined by the schema.",
+  "name": "base_event",
+  "namespace": "root",
+  "extends": "base_event",
+  "attributes": {
+    "raw_data": {
+      "group": "context"
+    },
+    "record_id": {
```

Comment:
record_id added as a new attribute
raw_data type is updated, hence declared here again